### PR TITLE
education user activities

### PR DESCRIPTION
**Pull request by VuTranAnh**
------------------------------------------
[o] Build Success!
------------------------------------------

[o] reek: success
[o] rspec: success
[o] rubocop: success
[o] scss-lint: success
[o] brakeman: success
[o] eslint: success
[o] bundle-audit: success
[o] rails_best_practices: success

### CODE REVIEW

File: app/models/concerns/activity.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Activity < PublicActivity::Activity
+  scope :load_by_current_user, ->user_id do
+    where(owner_id: user_id, owner_type: "User").includes :trackable, :owner
```
Comment (lequangcanh) [inline app/models/concerns/activity.rb]: User.name

File: app/views/education/users/_activities.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div class="tab-content clearfix" id="tab-activities">
+  <%= render_activity activities %>
+  <div class="center">
+    <%= paginate activities, remote: true, params: {tab: "tab-activities" } %>
```
Comment (lequangcanh) [inline app/views/education/users/_activities.html.erb]: bỏ space cuối

File: app/views/education/users/_posts.html.erb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 <div class="tab-content clearfix" id="tab-posts">
   <%= render "post", posts: posts %>
-  <%= paginate posts, remote: true, params: {tab: "tab-posts" } %>
+  <div class="center">
+    <%= paginate posts, remote: true, params: {tab: "tab-posts" } %>
```
Comment (lequangcanh) [inline app/views/education/users/_posts.html.erb]: bỏ space cuối

File: app/views/education/users/_projects.html.erb [hunk 4]
```diff
@@ -1,4 +1,6 @@
 <div class="tab-content clearfix" id="tab-projects">
   <%= render "project", projects: projects %>
-  <%= paginate projects, remote: true, params: {tab: "tab-projects" } %>
+  <div class="center">
+    <%= paginate projects, remote: true, params: {tab: "tab-projects" } %>
```
Comment (lequangcanh) [inline app/views/education/users/_projects.html.erb]: bỏ space cuối

File: app/views/public_activity/education_post/_create.html.erb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<li class="list-group-item user-show-activity">
+  <i class="icon-plus-sign"></i>
+  <span><%= activity.owner.name %></span>
+  <% if activity.trackable %>
+    <%= t ".created_post" %>
+    <%= link_to activity.trackable.title, activity.trackable %>
```
Comment (phamvanbk11) [inline app/views/public_activity/education_post/_create.html.erb]: dùng delegate đi em

File: app/views/public_activity/education_post/_update.html.erb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<li class="list-group-item user-show-activity">
+  <i class="icon-edit"></i>
+  <span><%= activity.owner.name %></span>
+  <% if activity.trackable %>
+    <%= t ".edited_post" %>
+    <%= link_to activity.trackable.title, activity.trackable %>
```
Comment (phamvanbk11) [inline app/views/public_activity/education_post/_update.html.erb]: dùng delegate

File: app/views/public_activity/education_project/_create.html.erb [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<li class="list-group-item user-show-activity">
+  <i class="icon-plus-sign"></i>
+  <span><%= activity.owner.name %></span>
+  <% if activity.trackable %>
+    <%= t ".created_project" %>
+    <%= link_to activity.trackable.name, activity.trackable %>
```
Comment (phamvanbk11) [inline app/views/public_activity/education_project/_create.html.erb]: dùng delegate

File: app/views/public_activity/education_project/_destroy.html.erb [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<li class="list-group-item user-show-activity">
+  <i class="icon-remove"></i>
+  <span><%= activity.owner.name %></span>
```
Comment (phamvanbk11) [inline app/views/public_activity/education_project/_destroy.html.erb]: dùng delegate

**Pull request merged**